src/test/java/com/openshift/internal/restclient/model/build/WebhookTriggerTest.java

```diff
@@ -29,13 +29,13 @@ public void setup(){
 	}
 	@Test
 	public void testGetWebhookUrlWhenResourceHasBaseURL() {
-		trigger = new WebhookTrigger(BuildTriggerType.generic, "secret101","foo","https://localhost:8443","v1beta1","test");
-		assertEquals("https://localhost:8443/osapi/v1beta1/buildConfigHooks/foo/secret101/generic?namespace=test", trigger.getWebhookURL());
+		trigger = new WebhookTrigger(BuildTriggerType.GENERIC, "secret101","foo","https://localhost:8443","v1beta1","test");
+		assertEquals("https://localhost:8443/osapi/v1beta1/buildConfigHooks/foo/secret101/Generic?namespace=test", trigger.getWebhookURL());
```

Comment:
Imho an exact expected match is what we want to assert, since the logic, that casing wouldnt matter is in the server and we dont want to assert this, right? Thus to me having the exact URL with uppercase in "Generic" is what I want to assert.